README.md

```diff
@@ -2,6 +2,15 @@
 
 Recipes for reproducing Analysis-Ready & Cloud Optimized (ARCO) ERA5 datasets.
 
+## Get Started with Colab Notebooks
+
+Explore the ERA5 dataset interactively with these notebooks:
+
+| Notebook | Description | Open in Colab |
+|----------|-------------|---------------|
+| [Surface Reanalysis Walkthrough](https://github.com/google-research/arco-era5/blob/main/docs/0-Surface-Reanalysis-Walkthrough.ipynb) | Learn how to work with surface-level ERA5 data | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google-research/arco-era5/blob/main/docs/0-Surface-Reanalysis-Walkthrough.ipynb) |
+| [Model Levels Walkthrough](https://github.com/google-research/arco-era5/blob/main/docs/1-Model-Levels-Walkthrough.ipynb) | Explore atmospheric model levels data | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google-research/arco-era5/blob/main/docs/1-Model-Levels-Walkthrough.ipynb) |
+
```

Comment:
After looking at the new README, I think this should be placed after the "Overview" section.